manual/manual_circuit.rst

```diff
@@ -817,6 +817,48 @@ The :py:class:`CircBox` construction is good for subroutines where the instructi
     print(circ.get_commands())
     print(circ.free_symbols())
 
+Clifford Tableaux
+=================
+
+The Clifford (a.k.a. stabilizer) fragment of quantum theory is known to exhibit efficient classical representations of states and unitaries. This allows for short descriptions that can fully characterise a state/unitary and efficient circuit simulation. Whilst the Clifford group can be characterised as the operations generated by `CX`, `H`, and `S` gates with qubit initialisation in the :math:`|0\rangle` state, it is also the largest group of operations under which the Pauli group is closed, i.e. for any tensor of Paulis :math:`P` and Clifford operation :math:`C`, :math:`CPC^\dagger` is also a tensor of Paulis.
+
+Any state in the Clifford fragment is uniquely identified by those tensors of Pauli operators that stabilize it (those :math:`P` such that :math:`P|\psi\rangle = |\psi\rangle`). These stabilizers form a group of size :math:`2^n` for an :math:`n` qubit state, but it is sufficient to identify :math:`n` independent generators to specify the group. If a Clifford gate :math:`C` is applied to the state, each generator :math:`P` can be updated to :math:`P' = CPC^\dagger` since :math:`C|\psi\rangle = CP|\psi\rangle = (CPC^\dagger)C|\psi\rangle`. We can therefore characterise each Clifford operation by its actions on generators of the Pauli group, giving us the Clifford tableau form. In ``pytket``, the :py:class:`UnitaryTableau` class uses the binary symplectic representation from Aaronson and Gottesman [Aaro2004]_.
```

Comment:
The jump from definition of Cliffords as _operations_ to description as _states_ needs a little explanation I think. Clifford state = Clifford operation applied to all-zero qubit state?

Reply:
This is somewhat implied by the inclusion of initialisation in |0> as part of the generators. Personally, I was in two minds to even have an explanation of stabiliser theory here, since the idea of the manual is that someone should already be familiar with quantum computing theory and we are just explaining how to map that into tket